infra/modules/aws-bulk/variables.tf

```diff
@@ -42,3 +42,8 @@ variable "source_kind" {
   default     = "hris"
   description = "Kind of the content to process"
 }
+
+variable "psoxy_basedir" {
+  type        = string
+  description = "the path where your psoxy repo resides"
```

Comment:
add default; although imho the various cases should be split out (path to jar, path to config, etc) - as there are legitimate reasons to support variations in layout